csfieldguide/chapters/content/en/coding-encryption/sections/storing-passwords-securely.md

```diff
@@ -170,6 +170,34 @@ In easy terms, what it is saying is that there are significantly fewer modificat
 Note that the estimates are based on trying to guess through a login system.
 With a leaked database, the attacker can test billions of passwords a second rather than just a few thousand.
 
+{panel type="project"}
+
+# Password guessing interactive
+
+Have a go at the following interactive.
+It requires you to guess the passwords of users.
+
+- Alice and Bob both used passwords that (by pure coincidence) are among the [top 25 most common passwords of 2018 according to SplashData](https://en.wikipedia.org/wiki/List_of_the_most_common_passwords).
+- Casey picked her password out of an english dictionary.
+  You might find hers a bit more challenging to figure out, although a computer would get it in a second.
+  If you are keen, you might like to do a dictionary attack on it yourself.
```

Comment:
The word `keen` seems a bit informal and slang-like. (Even though it might not technically be slang). `If you want to` probably sounds better.